test/verify/check-connection

```diff
@@ -497,8 +496,8 @@ class TestConnection(testlib.MachineCase):
         m.upload(["../tools/debian/tests/smoke"], "/tmp")
         m.execute("/tmp/smoke")
 
-        b.ignore_ssl_certificate_errors(ignore=True)
-        self.login_and_go("/system", tls=True)
+        b = self.new_browser(accept_insecure_ssl=True)
```

Comment:
Instead of having to construct a second Browser, what about using `provision` or something similar to control how the default browser is constructed?

Reply:
For example, I think the new browsers don't get the right `coverage` option value. Not sure if that matters, but it's potentially problematic.